src/vis/tooltip-manager.js

```diff
@@ -51,15 +51,24 @@ TooltipManager.prototype._addTooltipOverlay = function (layerView, layerModel) {
   }
 };
 
-TooltipManager.prototype._bindFeatureOverEvent = function (layerView, layerModel) {
+TooltipManager.prototype._bindFeatureOverEvent = function (layerView) {
   var tooltipView = layerView.tooltipView;
-  layerView.bind('featureOver', function (e, latlng, pos, data, layer) {
+  layerView.unbind('featureOver');
```

Comment:
be careful with this, you could be removing `featureOver` events from other objects.
You should do something like:
```
var f = function() {....}
layerView.unbind('featureOver', f);
layerView.bind('featureOver', f);
```